nucleus/core/kernel/src/main/java/fish/payara/enterprise/admin/commands/CreateJvmOption.java

```diff
@@ -0,0 +1,275 @@
+        }
+
+        String existingXms = bag.getStartingWith("-Xms");
+        if (existingXms != null) {
+            int xmsInConfig = JvmOptionBag.Duck.toMeg(existingXms, "-Xms");
+            int xmxGiven    = JvmOptionBag.Duck.toMeg(option, "-Xmx");
+            if (xmsInConfig > xmxGiven) { //i.e. domain.xml contains -Xms1g and you ask -Xmx512m to be set
+                String msg = lsm.getString("soft.xmx.smaller.than.xms", xmxGiven + " MB", xmsInConfig + " MB");
+                report.getTopMessagePart().addChild().setMessage(msg);
+                throw new IllegalArgumentException();
+            }
+        }
+    }
+
+    private void validateMin (JvmOptionBag bag, String option, ActionReport report) throws IllegalArgumentException {
+        if (!option.startsWith("-Xms")) {
+            return;
+        }
+
+        //now, opt is something like -Xms512m or -Xms2g, or -Xms=12 i.e. it may contain illegal characters
+        Pattern regex = Pattern.compile("-Xms((\\d)+[m|g|k|M|G|K]?)+");
+        boolean matches = regex.matcher(option).matches();
+        if (!matches) {
+            String msg = lsm.getString("soft.invalid.xms", option);
+            report.getTopMessagePart().addChild().setMessage(msg);
+            throw new IllegalArgumentException();
+        }
+
+        String existingXms = bag.getStartingWith("-Xms");
+        if (existingXms != null) {
+            String msg = lsm.getString("soft.xms.exists.singular", existingXms, option);
+            report.getTopMessagePart().addChild().setMessage(msg);
+        }
+        String existingXmx = bag.getStartingWith("-Xmx");
+        if (existingXmx != null) {
+            int xmxInConfig = JvmOptionBag.Duck.toMeg(existingXmx, "-Xmx");
+            int xmsGiven = JvmOptionBag.Duck.toMeg(option, "-Xms");
+            if (xmsGiven > xmxInConfig) { //i.e. domain.xml contains -Xms1g and you ask -Xmx512m to be set
+                String msg = lsm.getString("soft.xms.larger.than.xmx", xmsGiven + " MB", xmxInConfig + " MB");
+                report.getTopMessagePart().addChild().setMessage(msg);
+                throw new IllegalArgumentException();
+            }
+        }
+    }
+
+    /**
+     * Validates a JVM option that it is a valid option.
+     * <p>
+     * Valid options must either start with a {@code -} or <code>${ENV=}</code>
+     * @param bag Bag of JVM options that already exist
+     * @param option Option being added.
+     * @param report ignored
+     * @throws IllegalArgumentException If an invalid options is given
+     */
+    private void validate(JvmOptionBag bag, String option, ActionReport report) throws IllegalArgumentException {
+        if (!option.startsWith("-") && !option.startsWith("${ENV=")) {
+            String msg = lsm.getString("joe.invalid.start", option);
+            throw new IllegalArgumentException(msg);
+        }
+        if (bag.contains(option)) {
+            // setting an option that already exists is considered an error
```

Comment:
Probably outside the scope of this issue, but this doesn't seem to work unless the string matches exactly.
For example:
```
asadmin create-jvm-option -XX:MaxPermSize=512m
Successfully added JVM option: -XX:MaxPermSize=512m

asadmin create-jvm-option -XX:MaxPermSize=513m
Successfully added JVM option: -XX:MaxPermSize=513m

-XX:MaxPermSize=513m
remote failure: JVM option -XX:MaxPermSize=513m already exists in the configuration.
```